react-charts/area-chart/step5.md

```diff
@@ -1,34 +1,79 @@
-Now we'll add tooltips to the area chart...
+Now we will add axis labels to the area chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;Chart
   height={250}
   width={600}
-&gt;
-</pre>
-
-3) Add the following property to that section:
-
-<pre class="file" data-target="clipboard">
-containerComponent={
+  containerComponent={
   &lt;ChartVoronoiContainer
     constrainToVisibleArea
     labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`}
   /&gt;
 }
+&gt;
+</pre>
+
+3) <strong>Add the `padding` property to that section:</strong>
+
+In this case, give it the following values. More padding will be added to the bottom and left properties to accommodate the axis labels.
+
+- bottom: 75
+- left: 75
+- right: 50
+- top: 50
+
+It should look like this:
+
+<pre class="file" data-target="clipboard">
+padding={{
+  bottom: 75,
+  left: 75,
+  right: 50,
+  top: 50
+}}
 </pre>
 
-- The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered SVG
-- The `containerComponent` property takes a component instance which will be used to create a container element for standalone charts
-- The `ChartVoronoiContainer` is used to create voronoi overlays for charts, which are useful for attaching events to data points, such as tooltips
+4) <strong>Locate the code between the `<Chart>` and `<ChartGroup>`.</strong>
+
+<pre class="file">
+&lt;Chart&gt;
+&lt;/Chart&gt;
+
+&lt;ChartGroup&gt;
+&lt;/ChartGroup&gt;
+</pre>
+
+5) <strong>Between these components add the `<ChartAxis>` components.</strong>
+
+The `<Chart>` component displays an axis by default, but here you will add labels and grid lines to enhance the component.

```

Comment:
Let's end the first sentence at "default." Then let's start a new sentence: Add labels and grid lines here to enhance the component.